aleph/index/util.py

```diff
@@ -289,7 +289,27 @@ def configure_index(index, mapping, settings):
             "timeout": MAX_TIMEOUT,
             "master_timeout": MAX_TIMEOUT,
         }
-        config = es.indices.get(index=index).get(index, {})
+        res = es.indices.get(index=index)
+
+        if len(res) == 0:
+            # This case should never occur.
+            raise AlephOperationalException(
+                f"ES response does not contain data for index {index}"
+            )
+
+        if len(res) > 1:
```

Comment:
out of curiosity: how do you feel about merging these checks into a `if len != 1`?

Reply:
Fair point, I’ve pushed this.